bindata/etcd/openshift-recovery-tools

```diff
@@ -1,472 +0,0 @@
-#!/usr/bin/env bash
-export ETCDCTL_API=3
-
-ETCDCTL_WITH_TLS="$ETCDCTL --cert $ASSET_DIR/backup/etcd-client.crt --key $ASSET_DIR/backup/etcd-client.key --cacert $ASSET_DIR/backup/etcd-ca-bundle.crt"
-
-init() {
-  ASSET_BIN=${ASSET_DIR}/bin
-  if [ ! -d "$ASSET_BIN" ]; then
-    echo "Creating asset directory ${ASSET_DIR}"
-    for dir in {bin,tmp,shared,backup,templates,restore,manifests}; do
-      /usr/bin/mkdir -p ${ASSET_DIR}/${dir}
-    done
-  fi
-}
-
-# download and test etcdctl from upstream release assets
-dl_etcdctl() {
```

Comment:
looking at usage, this is the one shared function I think we have.  You can keep the one and the fixed env vars that are specific for these recovery scripts in this file.